app/controllers/notifications_controller.rb

```diff
@@ -17,7 +17,7 @@ def update
       return
     end
 
-    unless user.notifications_configured
+    if user.may_accept_notifications_screen?
```

Comment:
I like this naming better, but where is this defined?

Reply:
It's an aasm helper